content/docs/command-reference/push.md

```diff
@@ -181,11 +181,11 @@ One could do a simple `dvc push` to share all the data, but what if you only
 want to upload part of the data?
 
 ```dvc
-$ dvc push --with-deps matrix-train.p.dvc
+$ dvc push --with-deps test-posts
 
 ... Do some work based on the partial update
 
-$ dvc push --with-deps model.p.dvc
+$ dvc push --with-deps matrix-train
```

Comment:
same here - why do we need to change the stage name?

Reply:
I don't think `model` is a good stage name.